rust/chains/abacus-ethereum/src/inbox.rs

```diff
@@ -5,10 +5,11 @@ use std::{error::Error as StdError, sync::Arc};
 
 use async_trait::async_trait;
 use ethers::contract::abigen;
-use ethers::core::types::{H256, U256};
+use ethers::prelude::*;
```

Comment:
Just for my own learning: I'm curious how you normally think about preludes / wildcards in Rust in general? I'm no rustacean but my assumption was that it's a better practice to not use them if it's just a few types being imported. Also am curious what the effect there is on the compiler / the binary (if any) of bringing more things into scope than are actually used?

Reply:
I don't like wildcard imports with the exception of preludes. The point there is that it's a bunch of things you should "just have" in order to work with a given library. This is especially relevant because Trait functions are not available unless the trait is imported, even if the trait is implemented for a given thing.

It is important IMO to keep preludes to the minimum set of frequently used/QoL imports. Keep in mind even the rust lang has an implicit prelude with things like From and Into. (There is actually a way to disable it).